docs/pages/blog/2021.md

```diff
@@ -0,0 +1,185 @@
+The digitalization of the economy is accelerating, pushed by COVID-19.
+Developers and designers have been, more than ever, busy creating apps.
+
+We spent our year making it faster and easier for you to build great UIs and to delight your users with amazing UI experiences.
+Thank you for being part of this journey!
+
+## Growth
+
+It's only with your trust that we could achieve the following outcomes in 2021. Thank you!
+
+- 📦 From 6.4M to [10.8M](https://npm-stat.com/charts.html?package=%40material-ui%2Fcore&from=2019-11-30&to=2020-12-31) downloads per month (from 18% to [23%](https://docs.google.com/spreadsheets/d/1l5j3Xjtvm9XZtmb4ulLiWElQaXSlZlyCWT5ONrQMpBo/edit?usp=sharing) share of the React ecosystem).
+- 📈 From 4.6M to 6.0M unique visitors per year on the documentation.
+- ⭐️ From 63.8 to 74.0k stars, leave us [yours 🌟](https://github.com/mui-org/material-ui).
+- 👨‍👩‍👧‍👦 From 2,052 to [2,396](https://github.com/mui-org/material-ui/graphs/contributors) contributors.
+- 💰 Grew financial support by 2.9X in 2021, compared to 2020.
+- 🏢 From 6 to 14 full-time equivalent [team members](/about/).
+
+The numbers speak for themselves. We have reproduced the [traction we got in 2020](/blog/2020/#growth).
+
+## In review
+
+When we started 2021, we had recently released the first version of the MUI X data grid and were in the middle of the release of MUI v5.
+We had committed ourselves to fully execute these two major efforts:
+
+- On MUI Core, major releases have historically been efforts that span over a dozen months.
+  v5 was no different as great work takes time.
+  At the beginning of 2021, we had solved the most important aspect of v5, we had a new style engine to unlock a better styling and customization experience for developers.
+  But we still had most of the work ahead of us, we had to roll out the improvements to all our users.
+  With a large code and user base like MUI has, we knew it would take time.
+  For example, we might have had migrated a few components to the new style engine and fix all the issues that we could find, going to production is unforgiving.
+  Developers tend to surface all the cases that are overlooked in the first iteration with GitHub issues.
+- On MUI X, we had successfully sold our first Pro plan licenses and reached 100k downloads/month on the MIT version.
+  We had early signs from the community of a market fit for the product and that the open core model was the right one to execute our mission.
+  We were determined to solely focus on the data grid component, and demonstrate that our model was viable to build the best React data grid in the space.
+
+What might have been the biggest change in 2021 is the implications of a larger team for the organization structure.
+We have reached an inflection point at around 10 people.
+I could no longer look at all the GitHub issues and PRs.
+At that point, it was requiring >70h/week to do it correctly.
+One individual continuing doing it was not scaling.
```

Comment:
I didn't understand in the first time I read.

```suggestion
One individual doing that continuously was not scaling.
```

Reply:
I believe that your proposal changes the meaning. 

```suggestion
It was not scaling to have one individual aware of all PRs and GitHub issues.
```